### ISSUE TITLE
tab key is not working

### ISSUE THREAD

**Issue by eyalk5** [akiyosi/goneovim#99]
Hi,

tab key is not recognized. I am on macos high sierra. 
nvim. 0.5.0 goneovim 0.4.4 . 

I tried with `-u NONE`. going to input mode and pressing ^V and tab prints nothing. 

Thanks.

**Comment by akiyosi** [comment]
Hi

I'm using the same environment but the Tab key works.
What are your `~/.goneovim/settings.toml` settings?
goneovim have the external UI such as workspace sidebar, so the tab key may be robbed by the external UI if the mouse pointer is focused on the external UI.

**Comment by eyalk5** [comment]
```
[Tabline]
Visible = true
[MiniMap]
Visible = false
[SideBar]
 Visible = false
 DropShadow = false
 Width = 200
 AccentColor = "#5596ea" 

[Editor]

ExtTabline  = false
ExtCmdline = false
ExtPopupmenu = false
```

**Comment by eyalk5** [comment]
Hi,
do you have any suggestions? I'd appreciate it if there were debug logs to help identify the issue.

**Comment by akiyosi** [comment]
Sorry for the late reply.

This problem does not reproduce in my environment.(MacBook Pro 2013 Late with Mac OS High Sierra, and iMac 2019 with MacOS Mojave)
Also, the debug log function cannot be implemented immediately.

Is Tab not fully functional in your environment?
Or does it only work in certain scenes?

In addition, the keyboard type and keyboard language setting in the OS may be involved.

**Comment by eyalk5** [comment]
It doesn't work at all. It is functional in any other environment. 
It happens when I use the standard built-in keyboard. The keyboard is ANSI. 
Macpro late-2014 mac os high sierra. Thanks for testing. Let me know how to proceed. Want to move to your editor :)

**Comment by akiyosi** [comment]
I want to solve this problem.
This problem is fatal as an nvim GUI frontend.

Investigate problematic input processing and implement debug log output.

I want to check one, can You input `}` into goneovim in your environment? 
Also, does neovim-qt cause the same problem or not?

**Comment by eyalk5** [comment]
Well, it is solved in 0.4.5 somehow. neovim-qt doesn't have this issue.

**Comment by akiyosi** [comment]
Thanks for reply.
That was good, but strange as I haven't made any fixes related to this...
Anyway it was good.

**Comment by eyalk5** [comment]
Yeh, it doesn't really solved. It made a comeback. it sometimes happens and sometimes not, which makes me think it is related to keyboard configuration.

**Comment by akiyosi** [comment]
Hi :D
Goneovim will have improved input capabilities in the next release. 
I hope your problem will be improved in the next release.
Please wait a little longer.

**Comment by eyalk5** [comment]
Thanks. Still not solved in Goneovim v0.4.6. But at least I can mitigate it.

**Comment by eyalk5** [comment]
Works
